src/input.rs

```diff
@@ -16,9 +16,16 @@ pub fn keyval_to_input_string(in_str: &str, in_state: gdk::ModifierType) -> Stri
         debug!("keyval -> {}", in_str);
     }
 
+    // Use the Command key on macOS as Meta/Alt
+    let cmd_as_meta = env::var("NVIM_GTK_CMD_AS_META")
+        .map(|opt| opt.trim() == "1")
+        .unwrap_or(false);
+
     // CTRL-^ and CTRL-@ don't work in the normal way.
-    if state.contains(gdk::ModifierType::CONTROL_MASK) && !state.contains(gdk::ModifierType::SHIFT_MASK) &&
-        !state.contains(gdk::ModifierType::MOD1_MASK)
+    if state.contains(gdk::ModifierType::CONTROL_MASK)
```

Comment:
I actually realized maybe we should just make these changes outside of keyval_to_input_string() (sorry to make you write the unit tests! I don't think we actually needed those now ;_;), since it doesn't really make sense logically for an option to live in code that's just meant for converting gdk key values to nvim strings.

I think instead we should just come up with a function to modify the ModifierType bitmask higher up (e.g. in the callers of keyval_to_input_string()) instead, which will make it easier to pipe the setting from the RPC call through from the shell state.

Reply:
Do we want this remapping to apply to the entire window? I only really want it to apply to neovim, I think if we want to handle modifier keys ourselves for other reasons we should probably listen for the right ones. I guess if we only do this immediately before passing to neovim it's effectively the same, though it still feels a little more opaque to me.

Reply:
Yes - same way we handle other options like `Transparency` and `PreferDarkTheme`. And FWIW, the reason I think it makes more sense in some helper outside of keyval_to_input_string() is because then we'd have to carry around references to the shell state (or wherever we put the new option) in keyval_to_input_string(), which just feels a bit weird to me.

Reply:
I added the wrapper function, which also made testing quite a lot nicer/more possible, so I think that's indication you were right and it was probably a good idea to abstract that away into its own function.

However, I still feel that that logic should be performed in `convert_key` for two reasons: semantically "`convert_key`" seems like the fitting place to do these conversions, and because before then we don't really have easy mutable access to the modifier state since it's still inside an opaque `EventKey` struct. We could now pass a `&ModifierOptions` into `gtk_key_press` if we want to extract that out into global application state, which wouldn't be too bad. But honestly an env var just seems much simpler to me for this, mostly because it's really only a temporary workaround until actual "option-as-meta" can be implemented and this change has a much smaller footprint than adding a new rpc call at this time.

We'll probably want to do that extraction if/when option-as-meta is added, granted, but it doesn't feel worth it to me just yet because I fear that change may be more involved and will require manual keycode mapping. I understand if you disagree, and again I don't necessarily know if this is worth merging, but this feels like the best path for me for now.